janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchConstants.java

```diff
@@ -25,6 +25,9 @@
     public static final String ES_UPSERT_KEY = "upsert";
     public static final String ES_SCRIPT_KEY = "script";
     public static final String ES_INLINE_KEY = "inline";
```

Comment:
I would also remove `ES_INLINE_KEY` and `ES_INDEX_KEY` because they are not needed anymore

Reply:
done